src/librustdoc/passes/collect_intra_doc_links.rs

```diff
@@ -1079,7 +1078,9 @@ impl Disambiguator {
                 Namespace::MacroNS => "macro",
             },
         };
-        format!("{}@{}", prefix, path_str)
+
+        // FIXME: if this is an implied shortcut link, it's bad style to suggest `@`
```

Comment:
Oops, this should have been removed.

Reply:
Err, I meant to link https://github.com/rust-lang/rust/pull/75916/files#diff-2bcad7b16b56ef2ebb92f8e60ae33773R1308